packages/http-client-python/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "@typespec/http-client-python",
+  "version": "0.27.1",
+  "author": "Microsoft Corporation",
+  "description": "TypeSpec emitter for Python SDKs",
+  "homepage": "https://typespec.io",
+  "readme": "https://github.com/microsoft/typespec/blob/main/packages/http-server-python/README.md",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/typespec.git"
+  },
+  "bugs": {
+    "url": "https://github.com/microsoft/typespec/issues"
+  },
+  "keywords": [
+    "typespec",
+    "python"
+  ],
+  "type": "module",
+  "main": "dist/src/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/src/index.d.ts",
+      "default": "./dist/src/index.js"
+    },
+    "./testing": {
+      "types": "./dist/src/testing/index.d.ts",
+      "default": "./dist/src/testing/index.js"
+    }
+  },
+  "tspMain": "dist/src/index.js",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "scripts": {
+    "clean": "rimraf ./dist ./temp",
+    "build": "tsc -p ./emitter/tsconfig.build.json",
+    "watch": "tsc -p ./emitter/tsconfig.build.json --watch",
+    "test-official": "c8 mocha --forbid-only",
+    "lint": "eslint . --ext .ts --max-warnings=0",
+    "lint:fix": "eslint . --fix --ext .ts",
+    "install": "node ./eng/scripts/run-python3.cjs ./eng/scripts/install.py",
+    "prepare": "node ./eng/scripts/run-python3.cjs ./eng/scripts/prepare.py",
+    "regenerate": "node ./dist/eng/scripts/regenerate.js"
+  },
+  "files": [
+    "dist/**",
+    "!dist/test/**",
+    "generator/**",
+    "eng/scripts/**"
```

Comment:
aren't the scripts supposed to be for developement?

Reply:
the scripts are also used to build the package, so we had to add them as part of the package. This might mean that we shouldn't be putting `scripts/` inside of `eng/`